website/docs/guides/features-block.html.markdown

```diff
@@ -81,12 +81,16 @@ provider "azurerm" {
       purge_protected_items_from_vault_on_destroy             = true
     }
 
+    recovery_services_vault {
+      recover_soft_deleted_backup_protected_vm = true
+    }
+
     resource_group {
       prevent_deletion_if_contains_resources = true
     }
 
-    recovery_services_vault {
-      recover_soft_deleted_backup_protected_vm = true
+    storage {
+      data_plane_available = false
```

Comment:
Note: This is to be removed in 5.0 as we have a hard requirement to remove Data Plane dependencies from Resource Manager resources.